src/condor_scripts/annex/path-facility.sh

```diff
@@ -201,14 +204,22 @@ transfer_executable         = true
 arguments                   = ${JOB_NAME} ${COLLECTOR} ${LIFETIME} ${OWNERS} ${REQUEST_ID} \$(ClusterID)_\$(ProcID) ${CPUS} ${MEM_MB}
 
 transfer_input_files        = ${WELL_KNOWN_LOCATION_FOR_BINARIES}, ${WELL_KNOWN_LOCATION_FOR_CONFIGURATION}, token_file, password_file
-transfer_output_files       = \"\"
+# Transfer nothing back.
+# transfer_output_files       = \"\"
+# Debug: transfer back the log files.
+# transfer_output_files       = condor-9.5.4-1-x86_64_Rocky8-stripped/local/log
+# transfer_output_files       = condor-9.10.1-1-x86_64_AlmaLinux8-stripped/local/log
+transfer_output_files       = condor-9.11.1-0.602142-x86_64_AlmaLinux8-stripped/local/log
```

Comment:
Oof, but what are you gonna do... I suppose it would be pretty painful to try to fiddle with moving directories at the end of the job.

Reply:
(Resolving this note so that I can merge.)